src/Wallabag/CoreBundle/Helper/RuleBasedTagger.php

```diff
@@ -133,7 +133,7 @@ private function getRulesForUser(User $user)
     private function fixEntry(Entry $entry)
     {
         $clonedEntry = clone $entry;
-        $clonedEntry->setReadingTime($entry->getReadingTime() / $entry->getUser()->getConfig()->getReadingSpeed() * 200);
+        $clonedEntry->setReadingTime($entry->getUserReadingTime());
```

Comment:
note that before the set value is not rounded, while now with the new function it is.
wondering if this has an impact on if entries satisfies or not the rules.

Reply:
yep, it changes slightly the logic as we are rounding before comparing vs taking the float value without.
I think new behavior makes sense, as it aligns with UI behaviour like in the case below:
```
article reading time 17 min
user reading speed 400
article user reading time 9
```
This gives:
- before: a rule with >= 9 reading speed wouldn't have matched, despite the UI showing 9 so possibly user is confused
- after: a rule with >= 9 reading speed would now match the article

But let me know if you prefer keeping old behaviour

Reply:
Thanks for the detailed explanation! To me it seems in the scope of the bugfix so fine for me!